src/app/change-account-selector/change-account-selector.component.html

```diff
@@ -14,6 +14,7 @@
             ?  globalVars.loggedInUser.ProfileEntryResponse.Username
             : globalVars.loggedInUser.PublicKeyBase58Check
         }}</div>
+        <span *ngIf="globalVars.loggedInUser.isVerified" class="ml-1 d-inline-block align-center text-primary fs-12px"><i class="fas fa-check-circle fa-md align-middle"></i></span>
```

Comment:
```suggestion
        <span *ngIf="globalVars.loggedInUser.ProfileEntryResponse?.IsVerified" class="ml-1 d-inline-block align-center text-primary fs-12px"><i class="fas fa-check-circle fa-md align-middle"></i></span>
```

Reply:
`isVerified` needs to be capitalized and is a property of `ProfileEntryResponse` instead of `User`.